bitcoin/src/consensus/params.rs

```diff
@@ -88,8 +88,8 @@ impl Params {
                 bip66_height: 1,
                 rule_change_activation_threshold: 1916, // 95%
                 miner_confirmation_window: 2016,
-                pow_limit: Work::SIGNET_MIN,
-                pow_target_spacing: 10 * 60,            // 10 minutes.
+                pow_limit: Target::MAX,      // TODO: What should this be?
```

Comment:
https://github.com/bitcoin/bitcoin/blob/8105bce5b384c72cf08b25b7c5343622754e7337/src/kernel/chainparams.cpp#L348C40-L348C104

Reply:
Awesome, while writing a response I work it out, we just run the Core version through `Target::from_compact(t.to_compact_lossy())` like I did for the regtest one already, deeeer I'm slow sometimes :)

Added a `Target::SIGNET_MAX` as well as permalinks on all the consts to Core.